plugin/pkg/scheduler/algorithm/priorities/priorities_test.go

```diff
@@ -732,3 +732,164 @@ func TestBalancedResourceAllocation(t *testing.T) {
 		}
 	}
 }
+
+func TestImageLocalityPriority(t *testing.T) {
+	var mb int64 = 1000000
+	test_40_250 := api.PodSpec{
+		Containers: []api.Container{
+			{
+				Image: "gcr.io/40",
+			},
+			{
+				Image: "gcr.io/250",
+			},
+		},
+	}
+
+	test_40_140 := api.PodSpec{
+		Containers: []api.Container{
+			{
+				Image: "gcr.io/40",
+			},
+			{
+				Image: "gcr.io/140",
+			},
+		},
+	}
+
+	test_min_max := api.PodSpec{
+		Containers: []api.Container{
+			{
+				Image: "gcr.io/10",
+			},
+			{
+				Image: "gcr.io/2000",
+			},
+		},
+	}
+
+	node_40_140_2000 := api.NodeStatus{
+		Images: []api.ContainerImage{
+			{
+				RepoTags: []string{
+					"gcr.io/40",
+					"gcr.io/40:v1",
+					"gcr.io/40:v1",
+				},
+				Size: 40 * mb,
+			},
+			{
+				RepoTags: []string{
+					"gcr.io/140",
+					"gcr.io/140:v1",
+				},
+				Size: 140 * mb,
+			},
+			{
+				RepoTags: []string{
+					"gcr.io/2000",
+				},
+				Size: 2000 * mb,
+			},
+		},
+	}
+
+	node_250_10 := api.NodeStatus{
+		Images: []api.ContainerImage{
+			{
+				RepoTags: []string{
+					"gcr.io/250",
+				},
+				Size: 250 * mb,
+			},
+			{
+				RepoTags: []string{
+					"gcr.io/10",
+					"gcr.io/10:v1",
+				},
+				Size: 10 * mb,
+			},
+		},
+	}
+
+	tests := []struct {
+		pod          *api.Pod
+		pods         []*api.Pod
+		nodes        []api.Node
+		expectedList schedulerapi.HostPriorityList
+		test         string
+	}{
+		{
+			/*
+				Pod: gcr.io/40 gcr.io/250
+
+				Node1
+				Image: gcr.io/40 40MB
+				Score: (40M-23M)/97.7M + 1 = 1
+
+				Node2
+				Image: gcr.io/250 250MB
+				Score: (250M-23M)/97.7M + 1 = 3
+			*/
```

Comment:
As mentioned [here](https://golang.org/doc/effective_go.html#commentary) - line comments are the norm. Let's use line comments.